app/controllers/application_controller.rb

```diff
@@ -1,2 +1,18 @@
 class ApplicationController < ActionController::API
+  def not_found
+    render json: { error: 'not found' }
```

Comment:
Don't you think this can be better if moved into a controller concern? like:

```
# app/controllers/concerns/exception_handler.rb
module ExceptionHandler
  # provides the more graceful `included` method
  extend ActiveSupport::Concern

  included do
    rescue_from ActiveRecord::RecordNotFound do |e|
      json_response({ message: e.message }, :not_found)
    end
  end
end

and later can be included in the application controller in a more cleaner way

# app/controllers/application_controller.rb
class ApplicationController < ActionController::API
  include ExceptionHandler
end